src/oatpp/web/server/HttpConnectionHandler.cpp

```diff
@@ -86,7 +87,21 @@ void HttpConnectionHandler::handleConnection(const std::shared_ptr<oatpp::data::
 }
 
 void HttpConnectionHandler::stop() {
-  // DO NOTHING
+  /* Wait until all connection-threads are done but no longer than 1min */
+  auto startTime = std::chrono::system_clock::now();
+  auto timeout = std::chrono::minutes(1);
+
+  while(m_spawns->load() != 0) {
+    auto elapsed = std::chrono::system_clock::now() - startTime;
+    if(elapsed < timeout) {
+      std::this_thread::sleep_for(std::chrono::milliseconds(100));
+    } else {
+      if (m_spawns->load() > 0) {
+        OATPP_LOGW("[oatpp::web::server::HttpConnectionHandler::stop()]", "Timeout while waiting 1 minute for all connections to close. There are %ul connections still alive.", m_spawns->load());
```

Comment:
let's just block forever - until all threads are done.
Exiting this loop having running server threads will put an application into an unpredictable state.

Reply:
Ok!